images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -559,3 +559,28 @@ function Invoke-SBWithRetry {
         }
     }
 }
+
+function Get-GitHubPackageDownloadUrl {
+    param (
+        [string]$RepoOwner,
+        [string]$RepoName,
+        [string]$BinaryName,
+        [string]$Version,
+        [string]$UrlFilter
+    )
+
+    if ($Version -eq "latest") { 
+        $Version = "*" 
+    }
+    $json = Invoke-RestMethod -Uri "https://api.github.com/repos/${RepoOwner}/${RepoName}/releases?per_page=100"
```

Comment:
Does it make sense to move `per_page` to the params and set it to `100` by default just to stay on the safe side?